trigger.js

```diff
@@ -105,10 +108,30 @@ class TriggerHappy {
         this._executeTriggers(triggers);
     }
 
-    _onUpdateJournal(journal) {
-        if (journal.id === this._journalId || journal.name === this.journalName)
+    _onUpdateJournal(journal, update) {
+        if (update._id === this._journalId || journal.name === this.journalName)
             this._parseJournal();
     }
+
+    _onEndUpdate(scene, userId, update) {
+        if (update["x"] === undefined && update["y"] === undefined) return true;
+        let sceneTokens = scene.data.tokens;
+        let token = sceneTokens.find(t=>t._id === update._id);
+        if (token.hidden) return true; // hidden tokens don't trigger the trigger
+        let tokenNames = sceneTokens.map(t=>t.name); 
+        const triggers = this.triggers.filter(trigger => {
+            if (trigger.trigger.constructor.name !== "Token" || !tokenNames.includes(trigger.trigger.data.name) || trigger.trigger.data.name === token.name)
+                return false;
+            // let target = canvas.tokens.get(trigger.trigger.data._id);
+            let target = sceneTokens.find(token => token.name === trigger.trigger.data.name);
+            let tokenx = (update["x"] || token.x) + token.width * scene.data.grid / 2;

```

Comment:
use `const` instead of `let` since they aren't modified later.
Also, name it `tokenCenter` (or just `center`) since X means the X position, not the center point. (would have had to be `tokenX` instead of `tokenx` anyway, to conform with the coding style)